### ISSUE TITLE
Dark Mode 1.1

### ISSUE THREAD

**Issue by errorists** [status-im/status-react#10721]
# Problem
It seems that some people find the dark mode to not have enough contrast between the blue accent colour and dark backgrounds.
tl;dr this is a result of me trying to replace all blue values with a single colour. This colour couldn't be bright enough for punchy contrast because then outgoing message bubbles or notification badges would stop being readable with the white text on them.
To overcome that, we shouldn't change the accent blue on any components that have white text inside and then make the dark theme's blue brighter. 

## Implementation
Don't change the background colour on appearance change for

* [ ] outgoing messages (the dark blue ones you send)
* [ ] any Notification Badges

(anything else you can think of that has white text on blue background?, lmk)

* [ ] Update the dark theme's accent blue colour to a brighter `#758EF0`
* [ ] Update the keycard animation with values found [here](https://www.figma.com/file/9hkPcxsqDTWa9b9c49OX3X/Dark-Mode?node-id=177%3A4278)

## Acceptance Criteria
The above was implemented and I was able to test if the new blue is contrasty enough.

## Notes
Changes are already applied in [Dark Mode Figma](https://www.figma.com/file/9hkPcxsqDTWa9b9c49OX3X/Dark-Mode?node-id=5%3A6166)

Thanks for your patience while we work out the kinks of dark mode.

**Comment by hesterbruikman** [comment]
As discussed on Discord, to add here: Popover and Bottom sheet need more contrast

**Comment by churik** [comment]
Popups also need more contrast, as mentioned in #11789
